tests/e2e/raw_block_volume.go

```diff
@@ -118,13 +163,6 @@ var _ = ginkgo.Describe("raw block volume support", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		ginkgo.By(fmt.Sprintf("Deleting all statefulsets in namespace: %v", namespace))
```

Comment:
is this not required any more?

Reply:
no since this is added to defer call in the tc itself